packages/jest-config/src/set_from_argv.js

```diff
@@ -17,9 +17,10 @@ export default function setFromArgv(
   options: InitialOptions,
   argv: Argv,
 ): InitialOptions {
+  // $FlowFixMe: Seems like flow doesn't approve of string values
```

Comment:
This is really bad, so if someone more knowledgable about flow could have a look, that'd be great